app/src/renderer/windows/game/style.css

```
@@ -51,57 +55,205 @@
+.game-topnav__status-slot {
+  align-items: center;
+  color: var(--color-muted-foreground);
+  display: inline-grid;
+  flex: 0 0 auto;
+  height: 0.75rem;
+  justify-content: center;
+  line-height: 1;
+  margin-left: 0;
+  opacity: 0;
+  overflow: hidden;
+  place-items: center;
```

Stylelint (17.10.0)

[error] Overridden property "align-items" by shorthand "place-items" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)